sqrl-planner/sqrl-common/src/main/java/com/datasqrl/calcite/SqrlConfigurations.java

```diff
@@ -17,7 +17,7 @@ public class SqrlConfigurations {
       .withUpdateSetListNewline(false)
       .withIndentation(1)
       .withQuoteAllIdentifiers(true)
-      .withDialect(PostgresqlSqlDialect.DEFAULT)
+      .withDialect(ExtendedPostgresSqlDialect.DEFAULT)
```

Comment:
We need to be mindful to keep track of all the places where we hardcoded the dialect so we can generalize this soon. Nothing to change here right now. Just a general comment.